extensions/cornerstone/src/handleSegmentationStorage.js

```diff
@@ -1,8 +1,11 @@
-import * as dcmjs from "dcmjs";
+import * as dcmjs from 'dcmjs';
+import { api } from 'dicomweb-client';
 
-import OHIF from "@ohif/core";
-import cornerstone from "cornerstone-core";
-import cornerstoneTools from "cornerstone-tools";
+import OHIF from '@ohif/core';
+import cornerstone from 'cornerstone-core';
+import cornerstoneTools from 'cornerstone-tools';
+import DICOMWeb from '@ohif/core/src/DICOMWeb';
```

Comment:
@dannyrb Is this type of import across packages OK?

Reply:
ok I will change the approach :D